src/helpers/initialize.js

```diff
@@ -11,13 +11,14 @@ function initialize({ component, player, playerOpts }) {
   player.on('error', component.eventHandlers.onError);
   player.on('adPlay', component.eventHandlers.onAdPlay);
   player.on('adPause', component.props.onAdPause);
+  player.on('adComplete', component.props.onAdComplete);
+  player.on('adTime', component.eventHandlers.onTime);
   player.on('fullscreen', component.eventHandlers.onFullScreen);
   player.on('pause', component.props.onPause);
   player.on('play', component.eventHandlers.onPlay);
   player.on('mute', component.eventHandlers.onMute);
   player.on('playlistItem', component.eventHandlers.onVideoLoad);
   player.on('time', component.eventHandlers.onTime);
-  player.on('beforeComplete', component.props.onOneHundredPercent);
```

Comment:
This was from my initial experimentation of trying to retrieve the playhead position when the ad/video ends.  It was written in early on, and it should not have remained.  Removing it and restoring it to the original logic.